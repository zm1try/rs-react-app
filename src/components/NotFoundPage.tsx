export const NotFoundPage = () => {
  return <p>Route doesnt exist</p>;
};
