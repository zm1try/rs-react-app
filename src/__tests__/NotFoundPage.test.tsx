import { describe, it, expect } from 'vitest';
import { render } from '@testing-library/react';
import { NotFoundPage } from '@/components/NotFoundPage/NotFoundPage.tsx';

describe('NotFoundPage', () => {
  it('renders the not found message', () => {
    const { getByText } = render(<NotFoundPage />);
    expect(getByText('Route doesnt exist')).toBeInTheDocument();
  });

  it('matches the snapshot', () => {
    const { asFragment } = render(<NotFoundPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
