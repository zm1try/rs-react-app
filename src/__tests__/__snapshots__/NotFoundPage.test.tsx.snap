// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NotFoundPage > matches the snapshot 1`] = `
<DocumentFragment>
  <p>
    Route doesnt exist
  </p>
</DocumentFragment>
`;
